pkg/miner/miner_test.go

```diff
@@ -105,6 +105,70 @@ func TestMineStopResume(t *testing.T) {
 	)
 }
 
+func TestMineStopResumeReentrant(t *testing.T) {
```

Comment:
Can we add a description similar to other test cases in this file?

Reply:
https://github.com/keep-network/keep-core/pull/3174/commits/e144844dcbf8dfaee89d23428a0c476907f76fd0